src/test/kotlin/com/levi/featurespitter/FeaturespitterApplicationTests.kt

```diff
@@ -0,0 +1,13 @@
+package com.levi.featurespitter
+
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.context.SpringBootTest
+
+@SpringBootTest
+class FeaturespitterApplicationTests {
+
+	@Test
+	fun contextLoads() {
+	}
```

Comment:
There are no tests here, is this intentional?

How would you write the tests, and what would you test in a scenario where you had very limited time to write tests?

Reply:
I wanted to add the basic test cases for controller apis atleast, But could not write due to time constraint.

In limited time, I could have added following test scenarios - >

1. To get the total stores from database table and compare the count with api call result.
2. To check the the data fetched from end point and validate it with new database records.

In limited time, We can at-least check the data read from endpoint and data saved in database.

Reply:
👍